Formula/go.rb

```diff
@@ -34,51 +23,28 @@ class Go < Formula
     sha256 "5f570b6c7aa2d7caa6c715af6dce6fa30d7fbd5acc46fac8fbc3232270956f9e" => :mavericks
   end
 
-  devel do
-    url "https://storage.googleapis.com/golang/go1.7rc6.src.tar.gz"
-    version "1.7rc6"
-    sha256 "a289943548b838c7ef606a37836d1db080a3cb3c6df4e76456e23609b8505d05"
-
-    # Should use the last stable binary release to bootstrap.
-    # Not the case here because 1.6.3 is lacking a fix for an issue which breaks
-    # compile on macOS Sierra; in future this should share bootstrap with stable.
-    resource "gobootstrap" do
-      url "https://storage.googleapis.com/golang/go1.7rc6.darwin-amd64.tar.gz"
-      version "1.7rc6"
-      sha256 "ffe440747f7c663d7fc276b167ac630f921e66674c9952c97eed26fea9c8ac58"
-    end
-
-    go_version = "1.7"
-    resource "gotools" do
-      url "https://go.googlesource.com/tools.git",
-          :branch => "release-branch.go#{go_version}",
-          :revision => "a84e830bb0d2811304f6e66498eb3123ca97b68e"
-    end
-  end
-
   head do
     url "https://github.com/golang/go.git"
 
-    # Should use the last stable binary release to bootstrap.
-    # See devel for notes as to why not the case here, for now.
-    resource "gobootstrap" do
-      url "https://storage.googleapis.com/golang/go1.7rc6.darwin-amd64.tar.gz"
-      version "1.7rc6"
-      sha256 "ffe440747f7c663d7fc276b167ac630f921e66674c9952c97eed26fea9c8ac58"
-    end
-
     resource "gotools" do
       url "https://go.googlesource.com/tools.git"
     end
   end
 
   option "without-cgo", "Build without cgo"
   option "without-godoc", "godoc will not be installed for you"
-  option "without-vet", "vet will not be installed for you"
   option "without-race", "Build without race detector"
 
   depends_on :macos => :mountain_lion
 
+  # Should use the last stable binary release to bootstrap.
+  # More explicitly, leave this at 1.7 when 1.7.1 is released.
```

Comment:
We had this discussion recently 😉. It enables us to bootstrap with a build we know with a high degree of certainty works, so if there's build errors we know the errors are likely with the new release. Also gets people off our back around wasting their lives by building from source blah blah blah.